### ISSUE TITLE
Sentry - Error:: Couldn't delete file

### ISSUE THREAD

**Issue by qclmnt** [getsentry/sentry-cocoa#328]
_Platform:_
- [x]  iOS
- [ ]  tvOS
- [ ]  MacOS
- [ ]  watchOS

_Swift:_
- [x]  Yes -> If yes, which version? `4.4.0` and version `4.3.1`
- [ ]  No

_sentry-cocoa installed with:_
- [x] CocoaPods
- [ ] Carthage
- [ ] Manually

The version of sentry-cocoa:
**4.4.0** and version **4.3.1**

---
I have the following issue:

Description goes here ...

**Steps to reproduce:**
- `Client.shared?.send(event: event)` multiple time at the same moment (more than x100 times)

**Actual result:**
- Message well received on my sentry dashboard
- ERROR:
```ERROR: SentryCrashFileUtils.c (265): _Bool sentrycrashfu_readEntireFile(const char *const, char **, int *, int): Could not stat /var/mobile/Containers/Data/Application/25D32A8E-1C90-43BB-ACB9-70E6173F741C/Library/Caches/SentryCrash/Oqee/Reports/Oqee-report-00000000d200016d.json: No such file or directory
ERROR: SentryCrashC.c (274): char *sentrycrash_readReport(int64_t): Failed to load report ID d200016d
2019-09-18 10:44:22.292894+0200  Sentry - Error:: Couldn't delete file /var/mobile/Containers/Data/Application/25D32A8E-1C90-43BB-ACB9-70E6173F741C/Library/Caches/io.sentry/2475dd3dafb3921c0be38f8ae401279fd22244ea/events/1568796255.213789-62-2AA723B5-1133-4E72-9FE6-5164BDB2B942.json: Error Domain=NSCocoaErrorDomain Code=4 "Impossible de supprimer « 1568796255.213789-62-2AA723B5-1133-4E72-9FE6-5164BDB2B942.json »." UserInfo={NSFilePath=/var/mobile/Containers/Data/Application/25D32A8E-1C90-43BB-ACB9-70E6173F741C/Library/Caches/io.sentry/2475dd3dafb3921c0be38f8ae401279fd22244ea/events/1568796255.213789-62-2AA723B5-1133-4E72-9FE6-5164BDB2B942.json, NSUserStringVariant=(
    Remove
), NSUnderlyingError=0x287d499b0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
```

more than x100 times
* these multiple errors make my app freeze totally

**Expected result:**
- No Error in my consol, and not freeze

**Comment by whiskey** [comment]
This still happens in Sentry 5.0.5 but as it seems this might be caused by calling the initial setup (`SentrySDK.start(options:)`) multiple times.

**Comment by philipphofmann** [comment]
Thanks, @whiskey a d @qclmnt for pointing out the issue. We are aware of that and have plans to fix this withing version `6.0.0`. I can't give you an ETA though.

**Comment by philipphofmann** [comment]
This issue was solved 6.0.0 with https://github.com/getsentry/sentry-cocoa/pull/717, we know we could do better when capturing events in a tight loop, but at least the app only freezes slightly and this is not was this issue is originally about.
